lib/src/main/java/growthbook/sdk/java/GBFeaturesRepository.java

```diff
@@ -246,7 +245,15 @@ private void onSuccess(Response response) throws FeatureFetchException {
         }
     }
 
+    private void refreshSuccessfullyCallbacks(String featuresJson) {
+        this.refreshCallbacks
+                .forEach(featureRefreshCallback -> featureRefreshCallback.onRefresh(featuresJson));
+    }
 
+    private void refreshFailedCallbacks(Throwable throwable) {
```

Comment:
Can you rename this to `onRefreshFailed(Throwable throwable)`

Reply:
corrected)